clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -30,11 +35,37 @@ import org.ossreviewtoolkit.fossid.api.status.DownloadStatus
 import org.ossreviewtoolkit.fossid.api.status.ScanStatus
 
 import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.converter.jackson.JacksonConverterFactory
 import retrofit2.http.Body
 import retrofit2.http.GET
 import retrofit2.http.POST
 
 interface FossIdRestService {
+    companion object {
+        /**
+         * Create a FossID service instance for communicating with a server running at the given [url],
+         * optionally using a pre-built OkHttp [client].
+         */
+        fun create(
+            url: String,
+            client: OkHttpClient? = null
+        ): FossIdRestService {
+            val fossIdClient = (client ?: OkHttpClient()).newBuilder().build()
+
+            val mapper = JsonMapper()
+                .registerKotlinModule()
+                .setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE)
```

Comment:
Hint: After this commit, add a new commit that names this `jsonMapper` and makes it a public property of this companion object, and use this mapper also in the `EntityPostResponseBody<Any>.toList()` extension function, so we can get rid of the `api(project(":model"))` dependency in this module.